src/packs/nzism-v36-1022-20.ts

```diff
@@ -0,0 +1,845 @@
+/*
+ * This work is a derviate work based on the NZISM Conformance Paks, and cdk-nag from Amazon.com Inc. or its affiliates.
+ * Written by Andrew Frazer
+ * SPDX-License-Identifier: Apache-2.0
+ * The author of this nag-pak, makes no warranty or claims about its suitability or accuracy for any purpose, and accepts no liablity for its use.
+ */
```

Comment:
Appreciate the contribution, however the [standard disclaimer](https://github.com/cdklabs/cdk-nag/blob/main/src/packs/nist-800-53-r5.ts#L1-L4) should be used on each file

Reply:
Ok.